tests/fdr/test_fdr_minerRule.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python3
+
+from ffpack import fdr
+import numpy as np
+import pytest
+
+###############################################################################
+# Test minerDamageRuleNaive
+###############################################################################
+def test_minerDamageRuleNaive_twoPairs_scalarOutput():
+    fatigueData = [ [ 10, 100 ], [ 200, 2000 ] ]
+    calRst = fdr.minerDamageRuleNaive( fatigueData )
+    expectedRst = 0.2 
+    np.testing.assert_allclose( calRst, expectedRst )
+
```

Comment:
two empty lines between each function
three lines between the tests of different methods